### PR TITLE
Syncthing.Syncthing.x64: add version 1.23.4

### PR DESCRIPTION
**Pull request by torculus**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/101614)

### CODE REVIEW

**Comment by torculus** [comment]
Following up on [yesterday's discussion](https://github.com/microsoft/winget-pkgs/pull/101516), I have separated out Syncthing into a `Syncthing.Syncthing.x64` portable package. Upstream also released a bugfix yesterday, so that's excellent timing!

File: manifests/s/Syncthing/Syncthing.x64/1.23.4/Syncthing.Syncthing.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Syncthing.Syncthing
```
Comment (BrandonWanHuanSheng) [inline manifests/s/Syncthing/Syncthing.x64/1.23.4/Syncthing.Syncthing.locale.en-US.yaml:3]: Same Problem with above

Comment (BrandonWanHuanSheng) [inline manifests/s/Syncthing/Syncthing.x64/1.23.4/Syncthing.Syncthing.locale.en-US.yaml:3]: PackageIdentifier: Syncthing.Syncthing.x64

**Review by BrandonWanHuanSheng — CHANGES_REQUESTED** [review]
It look like the directory must change from manifests/s/Syncthing/Syncthing.x64/Syncthing.Syncthing.installer.yaml
manifests/s/Syncthing/Syncthing.x64/Syncthing.Syncthing.locale.en-US.yaml
manifests/s/Syncthing/Syncthing.x64/Syncthing.Syncthing.yaml
To Correct Directory 
manifests/s/Syncthing/Syncthing/x64/Syncthing.Syncthing.x64.installer.yaml
manifests/s/Syncthing/Syncthing/x64/Syncthing.Syncthing.x64.locale.en-US.yaml
manifests/s/Syncthing/Syncthing/x64/Syncthing.Syncthing.x64.yaml
You can press . button to open the editors. Go to the Navigation Panel to change the directory. You can commit and push to your PR after making those change on GitHub Pull Request.

**Comment by torculus** [comment]
Oops! Sorry about that. The new commit fixes this

**Comment by BrandonWanHuanSheng** [comment]
The correct file path @torculus 
manifests/s/Syncthing/Syncthing/x64/Syncthing.Syncthing.x64.installer.yaml
manifests/s/Syncthing/Syncthing/x64/Syncthing.Syncthing.x64.locale.en-US.yaml
manifests/s/Syncthing/Syncthing/x64/Syncthing.Syncthing.x64.yaml
There is no dots in the folder

**Comment by torculus** [comment]
Thanks @BrandonWanHuanSheng, I actually read your review carefully this time!

I did put the manifests in a `1.23.4` folder (i.e. `manifests/s/Syncthing/Syncthing/x64/1.23.4/`

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Instead of separating each architecture in multiple PackageIdentifiers, we can move NestedInstallerFiles to installer scope and modify the value of relative file path for each installer node as done in https://github.com/microsoft/winget-pkgs/pull/101625

**Comment by stephengillie** [comment]
Unfortunately, I'm not able to find the cause for this error.

**Comment by torculus** [comment]
It looks like #101625 passed validation and is a better solution than this branch. I'm happy to close this pull request (no merge) and defer to the other one. Thanks @mdanish-kh for your excellent work!

**Comment by mdanish-kh** [comment]
> It looks like #101625 passed validation and is a better solution than this branch. I'm happy to close this pull request (no merge) and defer to the other one. Thanks @mdanish-kh for your excellent work!

I added the architectures for the `1.23.3` version. You may still want to keep this PR open and update it accordingly for the newer `1.23.4` version

**Pull request closed without merging**